front-end/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:18 as build-step
+
+RUN mkdir -p /app
+
+WORKDIR /app
+
+COPY package*.json /app
+
+RUN npm install
+
+COPY . /app
```

Comment:
as `/app` is the workdir already, `COPY . .` should be here, as it does the same as `COPY . /app`